torchdata/dataloader2/communication/iter.py

```diff
@@ -166,7 +165,9 @@ def DataPipeBehindQueues(source_datapipe, protocol, full_stop=False, blocking_re
 
 class QueueWrapper(NonBlocking):
     """
-    Creates iter.DataPipe which reads data from the DataLoader.Queue
+    Creates an IterDataPipe which reads data from the DataLoader.Queue.
```

Comment:
This seems not the best description. It should be something like sending request and receiving data from subprocess queue.